scripts/install-kusion.sh

```diff
@@ -22,7 +22,7 @@ for arg in "$@"; do
     fi
 
     # base64 encode
-    content=$(base64 -w 0 "$path")
+    content=$(cat "$path" | base64 | tr -d '\n')
 
     # Build helm --set argument
```

Comment:
The change from 'base64 -w 0' to 'cat | base64 | tr -d '
'' is less efficient as it uses more processes and pipes. The original version using 'base64 -w 0' is more direct and performant.